build.cake

```diff
@@ -1,15 +1,12 @@
 //////////////////////////////////////////////////////////////////////
-// ARGUMENTS
+// ARGUMENTS & INITIALISATION
 //////////////////////////////////////////////////////////////////////
 
 var target = Argument("target", "Default");
-var configuration = Argument("configuration", "Debug");
```

Comment:
The msi uses the configuration to build - and it felt like it might slip people minds to package under a Release configuration! Making this 'Release' by default also makes it consistent with the framework repo